querybook/server/logic/metastore.py

```diff
@@ -320,6 +320,37 @@ def create_table_information(
     return table_information
 
 
+@with_session
+def create_table_warnings(
+    table_id,
+    warnings: tuple[DataTableWarningSeverity, str] = [],
+    commit=False,
+    session=None,
+):
+    """This function is used for loading table warnings from metastore.
+
+    For warnings from metastore, created_by will be None.
+    """
+    # delete all warnings without created_by from the table
+    session.query(DataTableWarning).filter_by(
```

Comment:
should we also be filtering by the table_id here?

Reply:
good catch!